kernel/core/datamanagement/dataaccess.cpp

```diff
@@ -1852,6 +1853,91 @@ static std::string tableMethod_anova(const std::string& sTableName, std::string
     return _kernel->getStringParser().createTempStringVectorVar(vRet);;
 }
 
+/////////////////////////////////////////////////
+/// \brief Realizes the "kmeansof()" table method.
+///
+/// \param sTableName const std::string&
+/// \param sMethodArguments std::string
+/// \param sResultVectorName const std::string&
+/// \return std::string
+///
+/////////////////////////////////////////////////
+static std::string tableMethod_kmeans(const std::string& sTableName, std::string sMethodArguments, const std::string& sResultVectorName)
+{
+    NumeReKernel* _kernel = NumeReKernel::getInstance();
+
+    std::string sCols = getNextArgument(sMethodArguments, true);
+    //sCols += "," + getNextArgument(sMethodArguments, true);
+
+    int nResults = 0;
+    _kernel->getMemoryManager().updateDimensionVariables(sTableName);
+    _kernel->getParser().SetExpr(sCols);
+    mu::value_type* v = _kernel->getParser().Eval(nResults);
+
+    if (nResults < 1)
+        throw SyntaxError(SyntaxError::TOO_FEW_COLS, sTableName + "().kmeansof()", ".kmeansof(", ".kmeansof(");

```

Comment:
This is not necessary to check, bc the parser will throw already an error in case the stack is empty.